src/helpers/stores/paths.js

```diff
@@ -1,21 +1,12 @@
+import config from '~helpers/configHelper';
 import { writable } from 'svelte/store';
 
 const PATHS_STORAGE_KEY = 'paths';
 
 function getPaths() {
   const storedPaths = window.localStorage.getItem(PATHS_STORAGE_KEY);
   if (storedPaths) return JSON.parse(storedPaths);
-
-  return fetch('./config.json')
-    .then((res) => res.json())
-    .then((json) => {
-      let commonPaths = json.COMMON_PATH.split(',');
-      return commonPaths.map((path) => path.trim()).filter(Boolean);
-    })
-    .catch((e) => {
-      console.error('Something went wrong while fetching config.json', e);
-      return [];
-    });
+  return config.COMMON_PATH
```

Comment:
Based on the other places you depend on config the return type for this method changed with this change which mean we don't have to async the init method on line 42.

Reply:
I cleaned up the remaining async/await stuff added in #502
Hope i got everything 🤞